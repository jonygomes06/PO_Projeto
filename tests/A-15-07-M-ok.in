6
4
Def,Ghi
4 Ghi Jlm,Joao
4
Ghi Jlm,Joao
0
0
