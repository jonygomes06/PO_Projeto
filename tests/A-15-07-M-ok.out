Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Menu de obras
1 - Mostrar obra
2 - Mostrar todas as obras
3 - Efectuar pesquisa de termos
4 - Mostrar todas as obras de um criador
5 - Alterar inventário de uma obra
0 - Sair
Escolha uma opção: Introduza o identificador do criador: Mostrar todas as obras de um criador: Operação inválida: O criador 'Def,Ghi' não existe.
Menu de obras
1 - Mostrar obra
2 - Mostrar todas as obras
3 - Efectuar pesquisa de termos
4 - Mostrar todas as obras de um criador
5 - Alterar inventário de uma obra
0 - Sair
Escolha uma opção: Número inválido!
Escolha uma opção: Introduza o identificador do criador: Mostrar todas as obras de um criador: Operação inválida: O criador 'Ghi Jlm,Joao' não existe.
Menu de obras
1 - Mostrar obra
2 - Mostrar todas as obras
3 - Efectuar pesquisa de termos
4 - Mostrar todas as obras de um criador
5 - Alterar inventário de uma obra
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 